### ISSUE TITLE
dirPath is empty, nothing to create / plugin not found

### ISSUE THREAD

**Issue by tlessard** [Azure/azure-container-networking#132]
**Is this a request for help?**:

Yes

**Is this an ISSUE or FEATURE REQUEST?** (choose one):

Issue

**Which release version?**:

1.04

**Which componenet (CNI/IPAM/CNM/CNS)**:

CNM

**Which Orchestrator and version (e.g. Kubernetes, Docker)**
Docker on Windows Server 2016 Enterprise with Containers (IaaS VM)

**What happened**:
Receiving an error when launching azure-vnet-plugin.exe and then am unable to create docker network.

**What you expected to happen**:
Unsure what the output is supposed to be, expected no errors and docker network creation success.

**How to reproduce it** (as minimally and precisely as possible):
[azure-vnet.log](https://github.com/Azure/azure-container-networking/files/1980565/azure-vnet.log)

[azure-vnet.json.txt](https://github.com/Azure/azure-container-networking/files/1980572/azure-vnet.json.txt)
***Note - renamed azure-vnet.json to azure-vnet.json.txt to allow me to attach to this issue.

 - Create Windows Server Enterprise with Containers Virtual Machine in Azure.
 - Create data disk and move docker ProgramData and storage to data disk
 - Execute azure-vnet-plugin.exe as Administrator, from either OS disk or data disk
   -   Receive an error:  "dirPath is empty, nothing to create."  The plugin begins listening on TCP 48080.  azure-vnet.json is created in same directory as EXE.  azure-vnet.txt is created in same directory also.
   -   Move azure-vnet.json to E:\ProgramData\docker\plugins directory but receive same error.
 - Execute this command:  C:\Windows\system32>docker network create --driver=azure-vnet --ipam-driver=aure-vnet --subnet=10.16.8.0/22 azure

   -   Receive this error:  Error response from daemon: plugin not found

**Anything else we need to know**:
Docker daemon is executing from non-standard location but this is not causing any other issues.  This may be operator error, I am unsure where plugin should be stored.

**Comment by tamilmani1989** [comment]
@tlessard - "dirPath is empty, nothing to create." -> This message can be ignored. I didn't any issue in the attached log also.  Also Please make sure that you run azure-vnet-plugin.exe from admin prompt. Can you share me the command that you used to create network?

**Comment by tlessard** [comment]
Hi, thanks for the quick response!  I had run azure-vnet-plugin.exe by right-clicking and running as administrator yesterday.  Based on your response I ran it again via an admin CMD shell and that appeared to fix the issue.  I was able to issue this command to create my network and it succeeded.

docker network create --driver=azure-vnet --ipam-driver=azure-vnet --subnet=10.16.8.0/22 azure

Thank you again
